staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -70,6 +70,10 @@ type GeneratableKeyCert struct {
 	// PairName is the name which will be used with CertDirectory to make a cert and key names
 	// It becomes CertDirector/PairName.crt and CertDirector/PairName.key
 	PairName string
+
+	// CipherSuites is the list of allowed cipher suites for the server.
+	// Values are from tls package constants (https://golang.org/pkg/crypto/tls/#pkg-constants).
+	CipherSuites string
```

Comment:
this should probably go in SecureServingOptions and be []string, tied to a string slice flag